test/publishing_api_test.rb

```diff
@@ -514,6 +514,195 @@ def content_item_for_content_id(content_id, attrs = {})
+        assert_raises(GdsApi::PublishingApi::NoLiveVersion) do
+          @api_client.get_live_content(@content_id)
+        end
+      end
+    end
+
+    describe "when an older version of the content item is live" do
+      before do
+        publishing_api
+          .given("a published content item exists with a draft edition for content_id: #{@content_id}")
+          .upon_receiving("a request to return the content item")
+          .with(
+            method: :get,
+            path: "/v2/content/#{@content_id}",
+            query: "locale=en",
+            headers: GdsApi::JsonClient.default_request_headers.merge(
+              "Authorization" => "Bearer #{@bearer_token}",
+            ),
+          )
+          .will_respond_with(
+            status: 200,
+            body: {
+              "content_id" => @content_id,
+              "document_type" => Pact.like("special_route"),
+              "schema_name" => Pact.like("special_route"),
+              "publishing_app" => Pact.like("publisher"),
+              "rendering_app" => Pact.like("frontend"),
+              "locale" => Pact.like("en"),
+              "routes" => Pact.like([{}]),
+              "details" => Pact.like({}),
+              "state_history" => { "1" => "published", "2" => "draft" },
+              "publication_state" => "draft",
+            },
+            headers: {
+              "Content-Type" => "application/json; charset=utf-8",
+            },
+          )
+          .upon_receiving("a request to return the live content item")
+          .with(
+            method: :get,
+            path: "/v2/content/#{@content_id}",
+            query: "locale=en&version=1",
+            headers: GdsApi::JsonClient.default_request_headers.merge(
+              "Authorization" => "Bearer #{@bearer_token}",
+            ),
+          )
+          .will_respond_with(
+            status: 200,
+            body: {
+              "content_id" => @content_id,
+              "document_type" => Pact.like("special_route"),
+              "schema_name" => Pact.like("special_route"),
+              "publishing_app" => Pact.like("publisher"),
+              "rendering_app" => Pact.like("frontend"),
+              "locale" => Pact.like("en"),
+              "routes" => Pact.like([{}]),
+              "public_updated_at" => Pact.like("2015-07-30T13:58:11.000Z"),
+              "details" => Pact.like({}),
+              "state_history" => { "1" => "published", "2" => "draft" },
+              "publication_state" => "published",
+            },
+            headers: {
+              "Content-Type" => "application/json; charset=utf-8",
+            },
+          )
+      end
+
+      it "returns the live content item" do
+        response = @api_client.get_live_content(@content_id)
+        assert_equal 200, response.code
+        assert_equal response["publication_state"], "published"
+      end
+    end
+
+    describe "when the content item has been unpublished" do
```

Comment:
This assertion feels a bit out of place since we don't have one about when content is published, just one about live. 

It feels like we'd either want to test both published and unpublished or we just want to test live as one assertion.

Reply:
I'll update the test description to match.